frontend/src/ui/Results/Results.tsx

```diff
@@ -64,6 +133,13 @@ const Results: React.FC<ResultsProps> = ({
             disabled={disabled}
             selected={showTreemap}
           />
+          <StyledSecondary
+            label="Export Chart"
+            testid="treechartPdf"
+            onClick={exportPDF}
+            // disabled={disabled}
+            // selected={showTreemap}
```

Comment:
Button is temporary. Will remove and hookup onclick to pdf export icon in treemap container in future.

Reply:
sean, now that I see Alex's comment, I agree. Let's add this disabled flag so that the button only shows in the case that the tree map is showing. We can decide if we want a different ui element to do the export later on.

Reply:
@Dev1nxavier once the latest code with the fix is in and the tests are passing I will merge it in. Just let me know. Thanks!